spec/files/test2.csv

```diff
@@ -0,0 +1,2 @@
+Title,Description,Price,Category

```

Comment:
Usually there is a directory: `spec/fixtures` with such files. Then you could have more specific directory for rewards and more informative names for files, eg:
```
spec/fixtures/rewards/rewards_import.csv
spec/fixtures/rewards/invalid_rewards_import.csv

```

Reply:
Understood! I have created folder fixtures and moved testing files there. I have also changed .csv files' names to more informative ;)